console/src/main.js

```diff
@@ -100,6 +234,9 @@ function openFileBrowser(path) {
     }
 }
 
+app.on('window-all-closed', function() {
+});
```

Comment:
Yep, I looked at it and was about to say we should remove and remembered it is important. Added.